feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt

```diff
@@ -0,0 +1,84 @@
+package tv.fw.feedintegration.recyclerview
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import java.util.UUID
+import tv.fw.common.feed.FeedResource
+import tv.fw.error.FwError
+import tv.fw.error.FwErrorListener
+import tv.fw.feedintegration.BuildConfig.FW_CHANNEL_ID
+import tv.fw.feedintegration.BuildConfig.FW_PLAYLIST_ID
+import tv.fw.feedintegration.R
+import tv.fw.videofeed.FeedItemClickListener
+import tv.fw.videofeed.FeedViewState
+import tv.fw.videofeed.FeedViewStateListener
+import tv.fw.videofeed.baseOptions
+import tv.fw.videofeed.viewOptions
+
+/**
+ * A fragment representing a list of Items.
```

Comment:
Now I know where that came from https://github.com/loopsocial/firework_android_sdk_examples/pull/26#discussion_r1084041227 😁